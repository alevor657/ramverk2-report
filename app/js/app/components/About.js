import React, { Component } from 'react';

class About extends Component {
    constructor(props) {
        super(props);
    }

    render() {
        return (
            <div className="container">
                <h1>About page</h1>
                <p>Denna sidan är skapat i samband med kuser ramverk2 som jag läser på BTH.</p>
                <a target="_blank" href="https://github.com/alevor657/ramverk2-report" rel="noopener noreferrer">Github</a>
                <div className="gallery-container">
                    <div>
                        <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/React-icon.svg/1200px-React-icon.svg.png"/>
                    </div>
                    <div>
                        <img src="http://mean.io/wp-content/themes/twentysixteen-child/images/express.png"/>
                    </div>
                    <div>
                        <img src="https://cdn-images-1.medium.com/max/1920/1*gdoQ1_5OID90wf1eLTFvWw.png"/>
                    </div>
                    <div>
                        <img src="http://sass-lang.com/assets/img/logos/logo-b6e1ef6e.svg"/>
                    </div>
                </div>
            </div>
        );
    }
}

export default About;
