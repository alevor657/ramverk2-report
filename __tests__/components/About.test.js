// About.react-test.js
import React from 'react';
import About from '../../app/js/app/components/About';
import renderer from 'react-test-renderer';

test('About page renders', () => {
    const component = renderer.create(<About/>);
    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();
});
