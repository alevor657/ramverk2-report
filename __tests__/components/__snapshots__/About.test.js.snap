// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page renders 1`] = `
<div
  className="container"
>
  <h1>
    About page
  </h1>
  <p>
    Denna sidan är skapat i samband med kuser ramverk2 som jag läser på BTH.
  </p>
  <a
    href="https://github.com/alevor657/ramverk2-report"
    rel="noopener noreferrer"
    target="_blank"
  >
    Github
  </a>
  <div
    className="gallery-container"
  >
    <div>
      <img
        src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/React-icon.svg/1200px-React-icon.svg.png"
      />
    </div>
    <div>
      <img
        src="http://mean.io/wp-content/themes/twentysixteen-child/images/express.png"
      />
    </div>
    <div>
      <img
        src="https://cdn-images-1.medium.com/max/1920/1*gdoQ1_5OID90wf1eLTFvWw.png"
      />
    </div>
    <div>
      <img
        src="http://sass-lang.com/assets/img/logos/logo-b6e1ef6e.svg"
      />
    </div>
  </div>
</div>
`;
